### DISCUSSION TITLE
Select Tab to see Cookies and then Perhaps Delete?

### DISCUSSION THREAD

**Discussion by deehill** [Cookie-AutoDelete/Cookie-AutoDelete#1195] [Ideas / Feature Requests]
Hi.

I've been looking over a number of Chrome/Firefox extensions dealing with Cookies. 

I'm interested in being able to "somehow" specify a given TAB, and then display the Cookies, and possibly then delete the cookies. This extension is the "closest" I've seen in that it apparently detects a given TAB once the TAB is deleted.

I'm curious, is there a way to modify the extension to do what I'm looking to do?

Any idea as to how long it might take to modify the extension? Any idea as to what you might charge for this modification?

Hit me up if you want to discuss a bit.

Thanks!

**Comment by kennethtran93** [comment]
All Cookies are stored by domain, not tab. You will not be able to see what cookies are set by a specific tab, especially if you open the same domain on multiple tabs. 

The way we keep track of tabs is mapping it to its current domain, but the mapping is only to check if its domain changed to see if we need to trigger cleanup on domain change. Handling the on tab close only removes its tab mapping plus trigger our auto cleanup if needed.

Hope this helps.
